pkg/ccl/backupccl/backup_job.go

```diff
@@ -860,6 +869,29 @@ func (b *backupResumer) Resume(ctx context.Context, execCtx interface{}) error {
 		if reloadBackupErr != nil {
 			return errors.Wrap(reloadBackupErr, "could not reload backup manifest when retrying")
 		}
+		// Re-load the job in order to update our progress object, which
+		// may have been updated since the flow started.
+		reloadedJob, reloadErr := p.ExecCfg().JobRegistry.LoadClaimedJob(ctx, b.job.ID())
```

Comment:
In general, we shouldn't need to reload the job from the job coordinator (i.e the node running resume). Check out how we avoid a job re-load in restore [here.](https://github.com/msbutler/cockroach/blob/dcd1638571e12f80da01130374bc2ffe58a14697/pkg/ccl/backupccl/restore_job.go#L212)

Reply:
the import code is provides a nice template as well https://github.com/cockroachdb/cockroach/pull/119708/commits/5e6ce265363538a21d49afb91393f52dc76a75e3